sdk/digitaltwins/azure-digitaltwins-core/tests/test_relationships.py

```diff
@@ -50,15 +50,24 @@ def _clean_up_models(self, client, *models):
     
     def _clean_up_relationships(self, client):
         for dt_id in [ROOM_DIGITAL_TWIN, FLOOR_DIGITAL_TWIN, BUILDING_DIGITAL_TWIN]:
-            for relationship in client.list_relationships(dt_id):
-                client.delete_relationship(
-                    dt_id,
-                    relationship['$relationshipId']
-                )
+            try:
```

Comment:
This change is a bit worrisome - why do we now need to swallow exceptions from these?